tests/validation/tests/v3_api/test_app.py

```diff
@@ -581,24 +582,29 @@ def create_project_client(request):
         rbac_data["users"][CLUSTER_MEMBER]["user"]
     user_token["user_c1_member"]["token"] = \
         rbac_data["users"][CLUSTER_MEMBER]["token"]
-    client, clusters = get_user_client_and_cluster_app()
+    # create a cluster
+    node_roles = [["controlplane"], ["etcd"],
+                  ["worker"], ["worker"], ["worker"]]
```

Comment:
Can we make this a 3 node cluster with 1 node - all roles and 2 worker nodes ?

Reply:
Done.